doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -219,14 +270,38 @@ If the sensor trigger fires the sensor change state to :c:enumerator:`SENSOR_STA
 Sensor trigger activation
 =========================
 
-The sensor trigger is activated and the sensor is put to sleep only if the values measured by the sensor do not deviate from the last sensor value by more than :c:member:`trigger.activation.threshold` for the period of time specified in :c:member:`trigger.activation.timeout_ms`.
+The sensor trigger is activated and the sensor is put to sleep if the values measured by the sensor do not deviate from the last sensor value by more than :c:member:`trigger.activation.threshold` for the period of time specified in :c:member:`trigger.activation.timeout_ms`.
 If the value measured by the sensor does not fit within the threshold, the last sensor value is updated and the sensor continues the sampling process.
 
 The sensor trigger activation type can be of the following type:
 
 * :c:enumerator:`ACT_TYPE_ABS` - Absolute deviation.
 * :c:enumerator:`ACT_TYPE_PERC` - Percentage deviation.
 
+Passive power management
+========================
+
+If :kconfig:`CONFIG_CAF_SENSOR_MANAGER_PM` is enabled sensors react to ``power_down_event`` and ``wake_up_event``.
+
+If a ``power_down_event`` comes while the sensor is in the :c:enumerator:`SENSOR_STATE_ACTIVE` state, the sensor state changes to :c:enumerator:`SENSOR_STATE_SLEEP` and sensor stops sampling.
+If the sensor has trigger functionality configured, the trigger would be activated.
+If no trigger is configured, sensor device state is changed to state configured in :c:member:`sensor_config.suspend_pm_state`.
+This happens only if the :c:member:`sensor_config.suspend_pm_state` is different than the default value (:c:enumerator:`PM_DEVICE_STATE_ACTIVE`).
+
+.. note::
+    |device_pm_note|
+
+If a ``wake_up_event`` comes while the sensor is in the :c:enumerator:`SENSOR_STATE_SLEEP` state, the sensor switches to :c:enumerator:`SENSOR_STATE_ACTIVE` and starts actively sampling.
+If the sensor supports trigger functionality, the trigger would be deactivated.
+
+Active power management
+=======================
+
+If both :kconfig:`CONFIG_CAF_SENSOR_MANAGER_ACTIVE_PM` is enabled and the sensor's trigger functionality is configured, the sensor can submit ``wake_up_event``.
```

Comment:
If there is any sensor that is ACTIVE the power state would be restricted to POWER_MANAGER_LEVEL_ALIVE. If all the sensors are sleeping and if any has trigger activated the power state would be restricted to POWER_MANAGER_LEVEL_SUSPENDED. Having all the sensors sleeping and none of them with trigger would allow any power state.

Reply:
Added this and a little more.